examples/miral-shell/miral-screencast.sh

```diff
@@ -39,7 +41,7 @@ if [ -e ${output} ]; then echo "Output exists, moving to ${output}~"; mv ${outpu
 while [ ! -e "${socket}" ]; do echo "waiting for ${socket}"; sleep 1 ;done
 
 if [ ! -v tempfile ]; then tempfile=$(mktemp); fi
-mirscreencast --size ${width} ${height} -m ${socket} -f ${tempfile}& mirscreencast_pid=$!
+${bindir}mirscreencast --size ${width} ${height} -m ${socket} -f ${tempfile}& mirscreencast_pid=$!
```

Comment:
I'd put the slash after `}` here.

Reply:
This approach was C&P from miral-app, but when "$(dirname $0)" == "" putting the slash after "}" results in "/mirscreencast" and the script fails.

(I forget which scenario leads to that case, but I do know I had to fix it in miral-app.)